bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,269 @@
+# Babylon Node Setup
+
+## Table of Contents
+
+1. [Install Babylon Binary](#install-babylon-binary)
+2. [Setup Node Home Directory and Configure](#setup-your-node-home-directory-and-configuration)
+4. [Create Keys](#create-keys)
+5. [Sync Node](#sync-node)
+6. [Get Funds](#get-funds)
+
+
+## Install Babylon Binary 
+
+1. Install [Golang 1.21](https://go.dev/dl)
+2. Verify installation:
+
+```shell
+go version
+```
+
+3. Clone and build Babylon:
+```shell
+git clone git@github.com:babylonlabs-io/babylon.git
+cd babylon
+git checkout bbn-test-5  
+make install
+```
+<!-- TODO: testnet tag to be defined -->
+This command does the following:
+- Builds the daemon
+- Compiles all the Go packages in the project
+- Installs the binary 
+- Makes the `babylond` command globally accessible from your terminal
+
+You should expect to see the following output:
+
+```shell
+# Build output will show:
+go install -mod=readonly -tags "netgo ledger mainnet" \
+    -ldflags '\
+        -X github.com/cosmos/cosmos-sdk/version.Name=babylon \
+        -X github.com/cosmos/cosmos-sdk/version.AppName=babylond \
+        -X github.com/cosmos/cosmos-sdk/version.Version=v0.13.0 \
+        -X github.com/cosmos/cosmos-sdk/version.
+        Commit=976e94b926dcf287cb487e8f35dbf400c7d930cc \
+        -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" \
+        -w -s' \
+    -trimpath ./...
+```
```

Comment:
I would remove this. I expect the flags to be changing with every new bbn version (e.g., `Commit`), which would make it confusing for the executor.